packages/bierzo-wallet/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Change this and explain this project will be for demonstrating the basic interaction with the chrome extension.

Reply:
Fixed by [14f6201](https://github.com/iov-one/ponferrada/pull/79/commits/14f6201854bd623d3fb74608509f7e5b7c587dc8)